ft/src/main/kotlin/org/octopusden/octopus/license/management/plugins/gradle/license/LicensePluginRunner.kt

```diff
@@ -56,6 +56,7 @@ fun gradleProcessInstance(init: TestGradleDSL.() -> Unit): Pair<ProcessInstance,
 
     val licenseManagementVersion = System.getenv().getOrDefault("license-management.version", "1.0-SNAPSHOT")
     val licenseMavenPluginVersion = System.getenv().getOrDefault("octopus-license-maven-plugin.version", OCTOPUS_LICENSE_MAVEN_PLUGIN_VERSION)
+    val octopusReleaseManagementVersion = System.getenv().getOrDefault("octopus-release-management.version", "1.0")
```

Comment:
Fail with clear message that either GRADLE_OCTOPUS_RELEASE_MANAGEMENT_PLUGIN_VERSION or octopus-release-management.version should be specified
Don't use not existing 1.0 as default